snap/snapcraft.yaml

```diff
@@ -0,0 +1,138 @@
+name: mycroft-precise
+adopt-info: mycroft-precise
+summary: Mycroft Precise
+description: Mycroft Precise, a wake word spotter.
```

Comment:
A lightweight, simple-to-use, RNN wake word listener, including all the tools to train your own custom wake word.

Mycroft Precise monitors an audio stream ( usually a microphone ) and when it recognizes a specific phrase it triggers an event. For example, by default users of the Mycroft Voice Assistant are using a Precise model trained to spot the phrase "Hey Mycroft". When Precise recognizes this phrase it puts the Mycroft Voice Assistant into command mode performing speech recognition on whatever is next said by the person using the device. 

Mycroft Precise is fully open source and can be trained to recognize any short-phrase or sound,  from a name to a cough. 

The default "Hey Mycroft" model is included so to test it out, try:
`mycroft-precise.listen /snap/mycroft-precise/current/hey-mycroft/hey-mycroft.pb`

#### USAGE

Running the listener:
- `mycroft-precise` - Alias for mycroft-precise.engine
- `mycroft-precise.engine` - Run a model on raw audio data from stdin
- `mycroft-precise.listen` - Run a model on microphone audio input
- `mycroft-precise.listen-pocketsphinx` - Run the PocketSphinx listener

Data collection:
- `mycroft-precise.collect` - Record audio samples for use with Precise
- `mycroft-precise.add-noise` - Create a duplicate dataset with added noise

Training:
- `mycroft-precise.train` - Train a new model on a dataset
- `mycroft-precise.train-generated` - Train a model on infinitely generated batches
- `mycroft-precise.train-incremental` - Train a model to inhibit activation by marking false activations and retraining
- `mycroft-precise.train-optimize` - Use black box optimization to tune model hyperparameters
- `mycroft-precise.train-sampled` - Train a model, sampling data points with the highest loss from a larger dataset

Evaluation and analysis:
- `mycroft-precise.test` - Test a model against a dataset
- `mycroft-precise.test-pocketsphinx` - Test PocketSphinx against a dataset
- `mycroft-precise.eval` - Evaluate a list of models on a dataset
- `mycroft-precise.calc-threshold` - Update the threshold values of a model for a dataset to make the sensitivity more accurate and linear
- `mycroft-precise.graph` - Show ROC curves for a series of models
- `mycroft-precise.simulate` - Simulate listening to long chunks of audio to find unbiased false positive metrics

Model conversion:
- `mycroft-precise.convert` - Convert wake-word model from Keras to TensorFlow

Reply:
Ake mentioned this might get pulled in from elsewhere, but we need to update the docs where ever it gets pulled from, hence this adds to that rewrite.

Reply:
Also it might not be a good idea to use the setup.py description since that can use mark-down in a way that the snap doesn't seem to handle in the description field.

Reply:
Did this but limited the markdown usage a bit to retain readability on the commandline.